src/explore-education-statistics-frontend/src/pages/_app.tsx

```diff
@@ -5,6 +5,8 @@ import Router from 'next/router';
 import React from 'react';
 import Helmet from 'react-helmet';
 import './_app.scss';
+import Link from '@frontend/components/Link';
+import { RegisterLinkRenderer } from 'explore-education-statistics-common/src/components/Link';
```

Comment:
`@common/components/Link`

Reply:
It's not being used across projects yet. Just within the common project.